src/core/trtserver2.h

```diff
@@ -304,6 +305,42 @@ TRTSERVER_EXPORT TRTSERVER_Error* TRTSERVER2_InferenceRequestOutputData(
 TRTSERVER_EXPORT TRTSERVER_Error* TRTSERVER2_InferenceRequestRemoveAllOutputs(
     TRTSERVER2_InferenceRequest* inference_request);
 
+///
+/// TRTSERVER2_ModelIndex
+///
+/// Object representing model indices.
+///
+
+/// Get the object representing the indices of all unique models in the
+/// model repository.
+/// \param server The inference server object.
+/// \param model_indices Returns the TRTSERVER2_ModelIndex object that
+/// is used to manage the lifecycle of the returned strings.
```

Comment:
Don't write a programmer description, write the user description. "Return the TRTSERVER2_ModelIndex object that holds the index of all models contained in the server's model repository(s)."